### DISCUSSION TITLE
Does anyone know why http://ui.tabix.io/ is not accessible？

### DISCUSSION THREAD

**Discussion by Tiakon** [ClickHouse/ClickHouse#48366] [Q&A] [answered]
I used an embedded Tabix server in clickhouse as a visual interface for editing sql. Now it is inaccessible. I wonder what happened to it and whether it will recover?

**Comment by mdonkers** [comment, marked as answer]
Hi,

It seems there is an issue with Tabix itself, since https://tabix.io/ or https://tabix.io/docs are unreachable as well.
Nothing on ClickHouse side should have changed that impacts this.

Unfortunately, ClickHouse is not responsible for https://tabix.io/ , so we'd need to ask you to try and reach out to them for support.

**Reply by Tiakon** [reply]
Ok, thanks for your answer
